staging/src/k8s.io/client-go/util/exec/exec.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package exec
+
+// ExitError is an interface that presents an API similar to os.ProcessState, which is
+// what ExitError from os/exec is.  This is designed to make testing a bit easier and
+// probably loses some of the cross-platform properties of the underlying library.
+type ExitError interface {
```

Comment:
I see. Can we move `ExitError` to the remotecmd package? (removing the copy in client-go/util/exec/exec.go and in kubernetes/util/exec/exec.go). In a followup PR is fine. There are not many references to ExitError:
```bash
$ git grep -l "CodeExitError" | grep -v pkg/util/exec
pkg/client/unversioned/remotecommand/v4.go
pkg/kubectl/cmd/run.go
pkg/kubectl/cmd/util/helpers_test.go
pkg/kubelet/remote/remote_runtime.go
test/e2e/framework/util.go
```